src/components/table/template.njk

```diff
@@ -0,0 +1,29 @@
+<table class="govuk-c-table {{ classes }}">
+  {% if options.caption %}
+  <caption class="govuk-c-table__caption {% if options.captionSize %} {{ options.captionSize }} {% endif %}">{{ options.caption }}</caption>
+  {% endif %}
+  {% if data.head %}
+  <thead class="govuk-c-table__head">
+    <tr class="govuk-c-table__row">
+    {% for item in data.head %}
+      <th class="govuk-c-table__header {% if item.type %} govuk-c-table__header--{{ item.type }} {% endif %}" {% if item.colspan %} colspan="{{ item.colspan }}" {% endif %} {% if item.rowspan %} rowspan="{{ item.rowspan }}" {% endif %} scope="col">{{ item.text }}</th>
+    {% endfor %}
+  </tr>
+  </thead>
+  {% endif %}
+  <tbody class="govuk-c-table__body">
+    {% for rows in data.rows %}
+    <tr class="govuk-c-table__row">
+    {% for cell in rows %}
+      {% if (loop.first) and (options.isFirstCellHeader) %}
+      <th class="govuk-c-table__header" scope="row"> {{ cell.text }}</th>
+      {% elseif loop.first %}
+      <td class="govuk-c-table__cell {% if cell.type %} govuk-c-table__cell--numeric {% endif %}" {% if cell.colspan %} colspan="{{ cell.colspan }}" {% endif %} {% if cell.rowspan %} rowspan="{{ cell.rowspan }}" {% endif %} scope="row">{{ cell.text }}</td>
```

Comment:
This is still hardcoded so if you set e.g. `'type': 'date'` you'll end up with the numeric class being applied - should be as with the headings which interpolate the type into the class name.

Reply:
missed it, ta